Bio/GenBank/Scanner.py

```diff
@@ -1867,7 +1867,7 @@ def _feed_misc_lines(self, consumer, lines):
                         consumer.origin_name(line)
                 if line.startswith("TSA "):
                     line = line[3:].strip()
-                    consumer.tgs(line)
```

Comment:
In hindsight that is irksome - I did have a quick look at the diff when you reported this, but couldn't see the problem and hadn't set down to test it yet.

Reply:
I stared at the code for quite some time and didn't see it. For some reason `consumer` doesn't throw an error if you call a nonexistent function on it. I also missed it when reviewing your initial pull request. 🤷‍♀